docs/cookbook/adding-new-attribute-to-an-entity.md

```diff
@@ -288,61 +288,79 @@ You can use [`Faker`](https://github.com/fzaninotto/Faker) to generate random nu
 +   /**
 +    * @var \Faker\Generator
 +    */
-+   private $faker;
++   protected $faker;
 
     /**
-     * @param \Shopsys\ShopBundle\DataFixtures\Demo\ProductParametersFixtureLoader $productParametersFixtureLoader
-     * @param \Shopsys\FrameworkBundle\Model\Product\ProductDataFactoryInterface $productDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Product\ProductFacade $productFacade
+     * @param \Shopsys\FrameworkBundle\Model\Product\ProductVariantFacade $productVariantFacade
      * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
      * @param \Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroupFacade $pricingGroupFacade
+     * @param \Shopsys\ShopBundle\Model\Product\ProductDataFactory $productDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Product\Parameter\ProductParameterValueDataFactory $productParameterValueDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Product\Parameter\ParameterValueDataFactory $parameterValueDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Product\Parameter\ParameterFacade $parameterFacade
+     * @param \Shopsys\FrameworkBundle\Model\Product\Parameter\ParameterDataFactory $parameterDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\PriceConverter $priceConverter
 +    * @param \Faker\Generator $faker  
      */
     public function __construct(
-        ProductParametersFixtureLoader $productParametersFixtureLoader,
-        ProductDataFactoryInterface $productDataFactory,
+        ProductFacade $productFacade,
+        ProductVariantFacade $productVariantFacade,
         Domain $domain,
--       PricingGroupFacade $pricingGroupFacade
-+       PricingGroupFacade $pricingGroupFacade,
+        PricingGroupFacade $pricingGroupFacade,
+        ProductDataFactoryInterface $productDataFactory,
+        ProductParameterValueDataFactory $productParameterValueDataFactory,
+        ParameterValueDataFactory $parameterValueDataFactory,
+        ParameterFacade $parameterFacade,
+        ParameterDataFactory $parameterDataFactory,
+        PriceConverter $priceConverter,
```

Comment:
the diff should have the adding of the coma at the end of the `PriceConverter` line as well

Reply:
I think that it is not necessary, because it will look like we are adding PriceConverter. People know they need to add coma :-)

Reply:
Well.. we have it with the coma in all of the places.. its just to make sure that you don't forget to add it